### PR TITLE
fix: update metadata with cloud tables reference

### PR DESCRIPTION
**Pull request by vncsna**
### Purpose

- Improve table metadata

### Description

- Use cloud tables data to retrieve metadata

### Checklist

- [ ] I have reviewed the code changes.
- [ ] I have tested the changes locally.
- [ ] I have updated the documentation if needed.
- [ ] I have added/modified tests to ensure the changes are valid.

### CODE REVIEW

File: bd_api/apps/api/v1/admin.py [hunk 1]
```diff
@@ -279,13 +279,15 @@ def get_number_of_columns(table, bq_table):
...
+        bucket = cs_client.get_bucket(table.source_bucket_name)
+
         if bq_table.num_bytes:
             return bq_table.num_bytes
 
         if bq_table.table_type == "VIEW":
             try:
                 file_size = 0
-                folder_prefix = f"staging/{table.dataset.db_slug}/{table.db_slug}"
+                folder_prefix = f"staging/{table.bq_slug}"
```
Comment (vncsna) [inline bd_api/apps/api/v1/admin.py:290]: @laura-l-amaral pode confirmar se `staging` é a pasta principal?

File: bd_api/apps/api/v1/models.py [hunk 2]
```diff
@@ -980,16 +971,17 @@ class Meta:
...
-        dataset_slug = self.dataset.db_slug
-        bucket_slug = self.source_bucket_name
-        return f"{bucket_slug}.{dataset_slug}.{table_slug}"
+    def bq_slug(self):
+        """Get the table slug used in Big Query"""
+        if cloud_table := self.cloud_tables.first():
+            project = cloud_table.gcp_project_id
+            dataset = cloud_table.gcp_dataset_id
+            table = cloud_table.gcp_table_id
+            return f"{project}.{dataset}.{table}"
```
[the remark this answers is not in the document]
Comment (laura-l-amaral) [reply bd_api/apps/api/v1/models.py:980]: não, o caminho do bucket sempre é o `basdosdados`

File: bd_api/apps/api/v1/admin.py (hunk 1 above)
Comment (laura-l-amaral) [reply bd_api/apps/api/v1/admin.py:290]: sim!

File: bd_api/apps/api/v1/models.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (vncsna) [reply bd_api/apps/api/v1/models.py:980]: okay, valeu

**Pull request merged**